docs/user-guide/standard_transformer.md

```diff
@@ -546,6 +548,20 @@ By range: 0 to 360 (For example wind directions)
 
 To learn more about cyclical encoding, you may find this page useful: [Cyclical Encoding](https://towardsdatascience.com/cyclical-features-encoding-its-about-time-ce23581845ca)
 
+### Autoload
+
+Autoload loads tables and variables that explicitly declared in the request or response interface. This operation only applicable for **upi_v1** protocol. Below is specification of autoload
+```yaml
+autoload:
+  tableNames:
+    - table_name_1
+    - table_name_2
+  variableNames:
+    - var_name_1
+    - var_name_2
+```
+`tableNames` and `variableNames` are fields that list of table name and variables that declared on request or response payload, if `autoload` part of `preprocess` it will try to load information from request payload, otherwise it will load from response payload.
```

Comment:
Rephrase 
```
`tableNames` and `variableNames` are fields that list table name and variables declaration. If `autoload` is part of `preprocess` pipeline, it will try to load those declared table and variables from request payload, otherwise it will load from model response payload.
```